content/gans_and_augmentation.qmd

```diff
@@ -4,9 +4,98 @@ toc-title: '![](../imgs/cover.jpg){width=240px}<br> <h3>Augmentation of image da
 
 # AI image generation II {#sec-GANS}
 
-## Generative Adversial Networks (GAN)
+In AI Image Generation I we mentioned GANs without going into details. In this chapter, we will take a closer look at them. We will also briefly touch on image augmentation. 
+
+## Generative Adversarial Nets (GAN)
+
+Generative Adversarial Nets, as first proposed by @goodfellowGenerativeAdversarialNetworks2014, are a class of generative models that can be used to generate new data samples from a given dataset. They consist of two components: a generator and a discriminator. The generator takes random noise as input and tries to produce realistic-looking data samples, while the discriminator takes data samples as input and tries to distinguish between real and fake samples. The generator and discriminator are trained simultaneously in a game-theoretic framework, with the goal of minimizing the difference between the distribution of real and fake samples. To use the authors own words:
+
+>"The generative model can be thought of as analogous to a team of counterfeiters, trying to produce fake currency and use it without detection, while the discriminative model is analogous to the police, trying to detect the counterfeit currency. Competition in this game drives both teams to improve their methods until the counterfeits are indistiguishable from the genuine articles."
+
+While the rest of the paper goes into mathematical depth and is not really recommendable for the casual reader, the basic concept behind it is surprisingly simple. The following figure illustrates the concept:
+
+![GAN architecture shown here for a model trained on the MNIST dataset, from @PyTorchGANBasic](../imgs/gan_architecture.png)
+
+The generator takes random noise as input and tries to produce realistic-looking data samples, while the discriminator takes data samples as input and tries to distinguish between real and fake samples. The generator and discriminator are trained simultaneously in a game-theoretic framework. The discriminators job is to maximize the probability of correctly classifying real images as real and fake images as fake. This is done by comparing the output of the discriminator for real images to 1 and for fake images to 0. The generator's job is to minimize the probability that the discriminator can correctly classify its outputs as fake. This is done by comparing the output of the discriminator for fake images to 1.
```

Comment:
This part is pretty close to the one in line 11, no?

> The generator takes random noise as input and tries to produce realistic-looking data samples, while the discriminator takes data samples as input and tries to distinguish between real and fake samples. The generator and discriminator are trained simultaneously in a game-theoretic framework

Reply:
upsi, da war wohl continue schneller als ich das wieder löschen konnte ;)